src/session-summaries/design-systems-part-2.md

```diff
@@ -0,0 +1,81 @@
+# DESIGN SYSTEMS (part 2)
+
+## WHAT TO LEARN
+
+### In this session on design sytems(part 2), the following are the things you would be learning:
+
+* **What a design system is and why you need it.**
+
+* **Benefits of having a structured design system.**
+
+* **Different levels of the design systems.**
+
+* **Aspects of design systems.**
+
+* **The things to consider before choosing a particular design system.**
+
+* **When to roll-out your own design system.**
+
+
+## Discussion panel
+
+* **Host**: Segun Ola 
```

Comment:
Valentino is a host, not a guest.

Add Front-matter to the top of the file:
```
----
hosts: Segun Ola, Valentino Ugbala
Guests: Inumidun Amao
summarized_by: your-name
summarized_by_social_media: url-to-your-prefered-social-media-profile
----
```

We'll do a detailed review, if necessary when you create the new PR as suggested.